backend/supergraph-config.yaml

```diff
@@ -0,0 +1,15 @@
+federation_version: 2
+subgraphs:
+  crystal_library:
+    routing_url: http://backend:80
```

Comment:
Do we have a way of ensuring these launch on the correct ports in our dev environment?